packages/variants/src/variants.ts

```diff
@@ -6,13 +6,13 @@ function makeThemeFromPartials(
   partialPalette: Partial<IPalette>,
   partialSemantic: Partial<ISemanticColors>
 ): ITheme {
-  return createTheme({
-    ...originalTheme,
-    ...{
-      palette: { ...originalTheme.palette, ...partialPalette },
-      semanticColors: { ...originalTheme.semanticColors, ...partialSemantic }
-    }
-  });
+  // Create variant palette
```

Comment:
you also need to duplicate this behavior in the strong variant which does it slightly differently